CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ### Added
 
+- `Select`: Changed the margin of the menu to 3px ([@JorenSaeyTL](https://github.com/JorenSaeyTL) in [#2008](https://github.com/teamleadercrm/ui/pull/2008))
+- `Popover`: Changed the Popover offset to 3px ([@JorenSaeyTL](https://github.com/JorenSaeyTL) in [#2008](https://github.com/teamleadercrm/ui/pull/2008))
+
```

Comment:
This should also be under the `Changed` section, not the `Added` section

Reply:
Done